src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/VSRestoreSettingsUtilities.cs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NuGet.Commands;
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.ProjectModel;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    public class VSRestoreSettingsUtilities
```

Comment:
LegacyPackageReferenceProject uses some of these methods. 
There's overlap between the logic of both project types and I want to avoid duplicate code now that we have extracted the utils for unit testing.